gradle/libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
-compose = "1.0.1-SNAPSHOT"
-composesnapshot = "7584688" # a single character = no snapshot
+compose = "1.1.0-SNAPSHOT"
```

Comment:
I know it works with `1.0.1`, but not with `1.0.0`. Only used `1.1.0` because we currently need the right nav snapshot and all versions that had what we needed came with `1.1.0-SNAPSHOT`.

Reply:
Ah, makes sense!